src/encoded/static/components/item-pages/components/ItemFooterRow.js

```diff
@@ -0,0 +1,315 @@
+'use strict';
+
+import React from 'react';
+import _ from 'underscore';
+import { FileEntryBlock } from '../../browse/components/FileEntryBlock';
+import { expFxn, Schemas } from './../../util';
+import { console, object } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
+import { StackedBlockTable, StackedBlock, StackedBlockList, StackedBlockName, StackedBlockNameLabel } from '@hms-dbmi-bgm/shared-portal-components/es/components/browse/components/StackedBlockTable';
+
+/**
+ * Reusable Component for showing Aliases, External References, etc.
+ * Shown at bottom of Item pages.
+ *
+ * @class ItemFooterRow
+ * @type {Component}
+ * @prop {Object} context - JSON representation of current Item object. Should be available through Redux store's context.
+ * @prop {Object} schemas - JSON representation of sitewide schemas.
+ * @prop {number} width - Used to define ExternalReferencesStackedTable's width
+ */
+export const ItemFooterRow = React.memo(function ItemFooterRow({ context, schemas, width }) {
+    const { aliases = [], actions = [], external_references = [], alternate_accessions = [] } = context || {};
```

Comment:
Codacy found an issue: ['aliases' is assigned a value but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5995039)